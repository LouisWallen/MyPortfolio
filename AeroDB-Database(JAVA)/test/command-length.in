set a
set a 7 8
get a
get b
LIST
list snapshot
snapshot
list keys
list ENTRIES
del a
list ENTRIES
list keys
list snapshot
get
get a
drop
drop 1
del
del b
purge
push c
push c 4
append
append c 6
pick
pick d 6
PLUCK
pluck i
pop
pop g
drop 6
ROLLBACK
CHECKOUT
snapshot 5
min
min h
max
max f
sum
sum g
len
len y
rev
rev d
uniq
uniq y
sort
sort u
diff
DIFF f g
INTER
inter d s
UNION
union d d
CARTPROD
cartprod g g
byyye
BYE
