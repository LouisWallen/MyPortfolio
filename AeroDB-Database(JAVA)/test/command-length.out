> Wrong Command!

> ok

> [7 8]

> no such key

> Wrong Command!

> Wrong Command!

> saved as snapshot 1

> a

> a [7 8]

> ok

> no entries

> no keys

> Wrong Command!

> Wrong Command!

> no such key

> Wrong Command!

> ok

> Wrong Command!

> no such key

> Wrong Command!

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> Wrong Command!

> Wrong Command!

> no such key

> no such snapshot

> Wrong Command!

> Wrong Command!

> Wrong Command!

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> no such key

> Wrong Command!

> bye
